style.css

```diff
@@ -437,4 +437,382 @@ input[type=email] {
   .fontawesome {
     display: none;
   }
+
+  .uber-navig {
+    display: flex;
+    flex-direction: column;
+    width: 345px;
+    height: 596px;
+    padding: 16px;
+    margin: 25px 24px;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    background-color: #fff;
+    justify-content: center;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    width: 1600px;
+  }
```

Comment:
- [ ] The page allows the user to scroll left, for the user experience (UX), let us fix this with overflow-x:hidden.
![scroll](https://user-images.githubusercontent.com/55635977/126771337-54d28364-4bd9-4351-bb8b-6ca4dbffa56b.png)